### ISSUE TITLE
Exported files contain more info in footer than ideal

### ISSUE THREAD

**Issue by maggie0002** [BookStackApp/BookStack#2526]
**Describe the bug**
When exporting files as HTML or PDF a footer is included: 

```
Revision #12 
Created Thu, Jan 14, 2021 2:17 AM by test user 
Updated Sun, Jan 31, 2021 10:50 PM by user2
```
The 'revision', 'test user' and 'user2' text in the example above contain a href to the relative pages on the BookStack site. I feel this is probably not of huge use to those with entirely public BookStack instances, but of concern to those who have generated content under a more restricted user group. The links expose the user and provide dead-end links to revision histories that won't be accessible. 

**Expected behavior**

I would suggest keeping the information, but removing the href from the text to allow for a wider range of use cases.

**Comment by ssddanbrown** [comment]
Thanks @maggie0002,

I agree, The value of those elements is likely lesser than the value of them be not linked. Marked for next release.

**Comment by ssddanbrown** [comment]
Thanks again @maggie0002, This has been addressed in 6a4b020dd88452099d9e1d77997fe46d64eb22e2 and will be part of the next feature release.
